tests/python/frontend/mxnet/test_forward.py

```diff
@@ -107,6 +107,14 @@ def test_forward_resnet():
         mx_sym = model_zoo.mx_resnet(18)
         verify_mxnet_frontend_impl(mx_sym)
 
+def test_forward_leaky_relu():
+    data = mx.sym.var('data')
+    data = mx.sym.concat(data, -data, dim=1)  # negative part explicitly
+    mx_sym = mx.sym.LeakyReLU(data)
+    verify_mxnet_frontend_impl(mx_sym, (1, 3, 100, 100), (1, 6, 100, 100))
+    mx_sym = mx.sym.LeakyReLU(data, act_type='leaky')
```

Comment:
All other existing Relu test cases seem to follow same pattern. Can we refactor them to a single function?

Reply:
The only leaky relu has two patterns to be applied.  Having two tests for it looks good to me.

Reply:
ok fine.